api/services/userService.js

```diff
@@ -0,0 +1,80 @@
+const {userDao} = require("../models");
+const jwt = require('jsonwebtoken');
+const axios = require('axios');
+
+const login = async (code) => {
+    const getToken = await axios.post("https://kauth.kakao.com/oauth/token",
+        {
+            grant_type: "authorization_code",
+            client_id: process.env.CLIENT_ID,
+            redirect_uri: "http://localhost:3000/auth/kakao/callback",
+            code: code
+        },
+        {
+            headers: {
+                'Content-Type': 'application/x-www-form-urlencoded;charset=utf-8'
+            }
+        });
+
+    const accessToken = getToken.data.access_token
+    console.log(`accessToken: ${accessToken}`);
+
+    const getKakaoUserData = await axios.get(
+        `https://kapi.kakao.com/v2/user/me`,
+        {
+            headers: {
+                Authorization: `Bearer ${accessToken}`,
+                'Content-Type': 'application/x-www-form-urlencoded',
+            },
+        }
+    );
+
+    const kakaoId = getKakaoUserData.data.id;
+    const {nickname} = getKakaoUserData.data.properties;
+    const {email} = getKakaoUserData.data.kakao_account;
```

Comment:
```suggestion
    const { id: kakaoId, properties: { nickname }, kakao_account: { email } } = getKakaoUserData.data;
```
getKakaoUserData.data 안에 객체 데이터들을 구조분해할당하여 한번에 작성될 수 있도록 해주세요.